src/deprecations.jl

```diff
@@ -1 +1,5 @@
 # deprecations
+@deprecate select!(r::ArrayReg, bit::Integer) select!(r, BitStr64{nactive(r)}(bit))
```

Comment:
I think the deprecation error can be more readable if we just writes our own here

Reply:
We probably Does not need to deprecate it anymore, since we don't have indexing problem that causing confusing here.

Reply:
yeah, remove this then? I'll merge after this one is resolved.